### DISCUSSION TITLE
Unpin Github Repository that I don't own

### DISCUSSION THREAD

**Discussion by KueTone** [community/community#84176] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Github

I recently was cleaning up my Github profile and wanted to unpin one of the repositories that was marked on my profile. For some reason when I click on Customize my Pins, Save Pins button is greyed out no matter what I choose. In addition, when I click into the repository, I am not able to see the Pin / Unpin button that should be to the right of Watch. Because of that, I can't unpin the repository. I attached some photos of the problem childs.

![image](https://github.com/community/community/assets/66387900/8222e359-b9ec-4054-a95b-cb3f81dfbe7a)
![image](https://github.com/community/community/assets/66387900/fd4a90ad-4a4b-4e97-b9f4-703f2d320d18)

**Comment by bonizario** [comment]
This issue might be due to a temporary glitch in GitHub's interface or a browser issue. Here are a few things you can try:

1. **Clear your browser cache and cookies**: Sometimes, old data stored in your browser can cause issues with web pages. Clearing your browser cache and cookies can often resolve these issues.

2. **Try a different browser or incognito mode**: There might be an issue specific to the browser you're using. Trying a different browser or using incognito mode can help determine if this is the case.

3. **Check for browser extensions that might be interfering**: Some browser extensions can interfere with web pages, especially ones that modify the page's JavaScript or CSS. Try disabling extensions to see if this resolves the issue.

4. **Wait and try again later**: If none of the above solutions work, it might be a temporary issue on GitHub's end. In this case, waiting a while and trying again later might resolve the issue.

If none of these solutions work, you might want to contact GitHub support for further assistance.
